rules/pypi/api_token.yml

```
@@ -0,0 +1,25 @@
+- id: sider.secrets.pypi.api_token
+  pattern:
+    regexp: |-
+      pypi-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{70,}
```

`goodchek pattern` shows a warning. Can we need a escape for `-` in character class?

```shellsession
$ ls
goodcheck.yml  lefthook.yml  LICENSE  package.json  package-lock.json  README.md  rules
$ goodcheck pattern
(...skip...)
sider.secrets.pypi.api_token:
/home/gnakagaw/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/goodcheck-3.0.3/lib/goodcheck/pattern.rb:34: warning: character class has '-' without escape: /pypi-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{70,}/
  - /pypi-AgEIcHlwaS5vcmc[A-Za-z0-9-_]{70,}/
(...skip...)
$
```

```suggestion
      pypi-AgEIcHlwaS5vcmc[A-Za-z0-9\-_]{70,}
```

---

Thank you!